### DISCUSSION TITLE
Why is magnetised not compatible with relax?

### DISCUSSION THREAD

**Discussion by CenTdemeern1** [ppy/osu#17681] [Q&A] [answered]
Magnetised is not the same as autopilot so I don't see the problem

**Comment by bdach** [comment, marked as answer]
because if you choose both then you get autoplay, essentially

**Reply by CenTdemeern1** [reply]
No because like I said it's not the same as autopilot

**Reply by CenTdemeern1** [reply]
You need to actually be close to the hitcircle for magnetised to make the hitcircle and your cursor overlap, unless you heavily turn up the strength in the customisation menu

**Reply by smoogipoo** [reply]
It is at full strength.

**Reply by bdach** [reply]
> unless you heavily turn up the strength in the customisation menu

that's the point

**Reply by CenTdemeern1** [reply]
but below that strength it is not the same as autoplay, so why not allow it then?

**Reply by smoogipoo** [reply]
We're not yet (and maybe never will be) able to disallow mod combinations based on settings.
